packages/interledger/src/index.ts

```diff
@@ -84,3 +92,36 @@ if (!module.parent) {
     logger.error(errInfo)
   })
 }
+
+async function _setupConnector(): Promise<Rafiki> {
+  const tbClient = createClient({
+    cluster_id: Config.tigerbeetleClusterId,
+    replica_addresses: Config.tigerbeetleReplicaAddresses
+  })
+  const accountsService = new AccountsService(tbClient, Config, logger)
+
+  const pricesUrl = process.env.PRICES_URL // optional
+  const pricesLifetime = +(process.env.PRICES_LIFETIME || 15_000)
+  const ratesService = createRatesService({
+    pricesUrl,
+    pricesLifetime,
+    logger: logger
+  })
+
+  return createConnectorService({
+    redis,
+    logger,
+    ratesService,
+    accountsService
+  })
+}
+
+async function _setupAdminApi(): Promise<ApolloServer> {
+  const tbClient = createClient({
+    cluster_id: Config.tigerbeetleClusterId,
+    replica_addresses: Config.tigerbeetleReplicaAddresses
+  })
+  const accountsService = new AccountsService(tbClient, Config, logger)
+
+  return createAdminApi({ accountsService })
```

Comment:
Why not use the same `accountsService` for both the admin api and the connector?

Reply:
Mainly because I didn't want to clutter the start function with creating a bunch of services. But I'm happy to change it.

Reply:
Are there any drawbacks to having multiple tigerbeetle clients in the same process?

Reply:
Yeah probably, might just be easier not to test it.